README.md

```diff
@@ -257,6 +257,39 @@ DTM_IGNORED_MIGRATIONS = {
 
 And we won't complain about them.
 
+Or you can completely ignore entire app:
+
+```python
+# settings.py
+
+DTM_IGNORED_MIGRATIONS = {
+    ('dependency_app', '*'),
+    ('another_dependency_app', '*'),
+}
+```
+
+Or if you split your third party apps separately, you can go even further:
```

Comment:
This can be removed into in-code docstrings.

Reply:
Yep, I agree it's quite redundant.